src/rpc/net.cpp

```diff
@@ -960,7 +960,8 @@ static RPCHelpMan addpeeraddress()
         RPCResult{
             RPCResult::Type::OBJ, "", "",
             {
-                {RPCResult::Type::BOOL, "success", "whether the peer address was successfully added to the address manager"},
+                {RPCResult::Type::BOOL, "success", "whether the peer address was successfully added to the address manager table"},
+                {RPCResult::Type::STR, "error", /*optional=*/true, "error description, if the address could not be added (and success is false)."},
```

Comment:
In edc8cb64f9aae6939183de276828991243927917: Is it necessary the "(and success is false)"? Isn't it intuitive?